examples/how_to/assess_spotfindingresults.py

```diff
@@ -0,0 +1,88 @@
+"""
+.. _howto_spotfindingresults:
+
+Assessing :py:class:`.SpotFindingResults`
+=========================================
+
+Purpose of this tutorial:
+
+* Deciding between spot-based or pixel-based decoding
+* Choosing a :py:class:`.FindSpotsAlgorithm`
+* Tuning a :py:class:`.FindSpotsAlgorithm`
+
+Although it is not necessary to visualize spots found by the :py:class:`.FindSpotsAlgorithm` before
+decoding every field of view in your data, it can be a useful step when building an image
+processing pipeline. Visually assessing the detected spots will ensure the spot-based decoding
+approach and :py:class:`.FindSpotsAlgorithm` you chose is optimized for your data. To learn more
+about how spots can be found and decoded in starfish see :ref:`section_finding_and_decoding`.
+
+There are two methods for viewing spots. The first is to access the :py:class:`.SpotAttributes`
+of a selected ImageSlice and add it as points to the napari viewer. The second is to use a
```

Comment:
It is actually, we call an entry in SpotFiindinigResults `PerImageSliceSpotResults`

Reply:
Yeah, but someone reading the docs may not know that.

Reply:
Sounds like something that should be added to the glossary then?

Reply:
Added ImageSlice to glossary and linked to it from here. Other parts of the docs refer to an ImageSlice as an image volume, which is more intuitive to me because it implies the x,y,z shape.